cypress/support/api/circulation.js

```diff
@@ -101,6 +101,19 @@ Cypress.Commands.add('getNoticePolicy', (searchParams) => {
     });
 });
 
+Cypress.Commands.add('getLoanPolicy', (searchParams) => {
+  cy.okapiRequest({
+    method: REQUEST_METHOD.GET,
+    path: 'loan-policy-storage/loan-policies',
+    searchParams,
+  })
+    .then(policy => {
+      Cypress.env(CY_ENV.LOAN_POLICIES, policy.body.loanPolicies);
```

Comment:
Let's try to avoid using of env for Api calls
I understand that we have a lot of same things already
But it is not good practice